Vagrantfile

```diff
@@ -298,6 +310,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
                   ENV["CONTIV_CLUSTER_STORE"] || "etcd://localhost:2379",
                   ENV["CONTIV_DOCKER_VERSION"] || docker_version,
                   ENV['CONTIV_NODE_OS'] || "",
+                  ENV['CONTIV_V2PLUGIN_NAME'] || "contiv/v2netplugin:0.1",
```

Comment:
Why use a separate versioning scheme for v2netplugin container image? Shouldn't we use the same version as what is in netplugin/version/CURRENT_VERSION?